index.md

```diff
@@ -58,104 +58,105 @@ This page introduces guidelines and other standards related to web accessibility
 
-- [[Accessibility Conformance Testing (ACT) Overview]](/standards-guidelines/act/) &mdash; ACT establishes and documents rules for testing the conformance of web content to accessibility standards.
+- [[Resumen de Accessibility Conformance Testing (ACT)]](/standards-guidelines/act/) &mdash; ACT establece y describe normas para comprobar la conformidad del contenido web con los estándares de accesibilidad.
 
-- [[Evaluation and Report Language (EARL) Overview]](/standards-guidelines/earl/) &mdash; EARL is a machine-readable format for expressing test results.
+- [[Resumen de Evaluation and Report Language (EARL)]](/standards-guidelines/earl/) &mdash; EARL es un formato para ser leído por aplicaciones que sirve para expresar los resultados de las pruebas.
 
-Additional resources related to evaluation are listed in the [[Evaluating Web Accessibility Overview]](/test-evaluate/), including:
+Recursos adicionales relacionados con la evaluación se pueden encontrar en el [[resumen de evaluación de la accesibilidad web]](/test-evaluate/), incluyendo:
 
-- [[WCAG-EM Overview: Website Accessibility Conformance Evaluation Methodology]](/test-evaluate/conformance/wcag-em/) &mdash; WCAG-EM is an approach for determining how well a website conforms to Web Content Accessibility Guidelines (WCAG).
+- [[Resumen de WCAG-EM: Website Accessibility Conformance Evaluation Methodology]](/test-evaluate/conformance/wcag-em/) &mdash; WCAG-EM es una aproximación para determinar cuánto se ajusta un sitio web a las pautas de accesibilidad para el contenido web (WCAG).
 
-### Personalization
+### Personalización
 
-[[Personalization Overview]](/personalization/) &mdash; Personalization involves tailoring the user experience to meet the needs and preferences of the individual user. Content authors can use personalization standards to provide a default design and enable user personalization with minimal work.
+[[Resumen sobre la personalization]](/personalization/) &mdash; La personalización consiste en ajustar la experiencia del usuario a fin de cubrir las necesidades y preferencias individuales. Los autores de contenido pueden usar estándares de personalización para proporcionar un diseño por defecto y permitir la personalización del usuario con un esfuerzo mínimo.
 
-## Other Areas of W3C WAI work
+## Otras áreas del trabajo de W3C WAI
 
-- [[Mobile Accessibility at W3C]](/standards-guidelines/mobile/)
+- [[Accesibilidad móvil en W3C]](/standards-guidelines/mobile/)
 
-- [[Cognitive Accessibility at W3C]](/cognitive/)
+- [[Accesibilidad cognitiva en W3C]](/cognitive/)
 
-## Additional Information {#moreinfo}
+## Información adicional {#moreinfo}
 
-- [List of all W3C accessibility-related Standards ("W3C Recommendations") and Working Group Notes](https://www.w3.org/TR/#tr_Accessibility__All_)
+- [Listado de todos los estándares W3C relacionados con la accesibilidad ("W3C Recommendations") y notas de los grupos de trabajo](https://www.w3.org/TR/#tr_Accessibility__All_)
```

Comment:
- `Estándares`must be capitalized.
- `Recomendaciones` in Spanish.
- `Notas de los Grupos de Trabajo`